tests/wp-unit/tests/include/Core/Posts/Metabox/DeletePostsByCategoryMetaboxTest.php

```diff
@@ -305,26 +245,10 @@ public function test_for_delete_posts_older_than_x_days() {
 		$cat2 = $this->factory->category->create( array( 'name' => 'cat2' ) );
 
 		// Assign the cat1 and cat2 to post1 and post2
-		$post1 = $this->factory->post->create( array( 'post_title' => 'post1', 'post_status' => 'publish', 'post_category' => array( $cat1 ) ) );
-		$time = current_time('mysql');
-		// Update post1 publish date.
-		wp_update_post(
-			array (
-			'ID'            => $post1,
-			'post_date'     => $day_post,
-			'post_date_gmt' => $day_post,
-			)
-		);
+		$post1 = $this->factory->post->create( array( 'post_title' => 'post1', 'post_status' => 'publish', 'post_category' => array( $cat1 ), 'post_date'     => $day_post, 'post_date_gmt' => $day_post ) );
```

Comment:
You don't have to set `post_date_gmt`. If you are setting it, then you have to convert the date into GMT timezone and set it.